tests/handlers/logging/logging_tests.py

```diff
@@ -245,3 +247,47 @@ def test_arbitrary_object(logger):
     event = logger.client.events[ERROR][0]
     assert "param_message" in event["log"]
     assert event["log"]["param_message"] == "['a', 'list', 'of', 'strings']"
+
+
+def test_logging_filter_no_span():
+    requests_store = Tracer(lambda: [], lambda: [], lambda *args: None, Config(), None)
+    requests_store.begin_transaction("test")
+    f = LoggingFilter()
+    record = logging.LogRecord(__name__, logging.DEBUG, __file__, 252, "dummy_msg", [], None)
+    f.filter(record)
+    assert record.elasticapm_transaction_id
```

Comment:
`requests_store.begin_transaction("test")` returns the `Transaction` object, so you could assert that the IDs are identical instead of "just" their existence. Same in the other tests below

Reply:
Done.